public/app/features/org/OrgDetailsPage.test.tsx

```diff
@@ -1,4 +1,5 @@
-import { shallow } from 'enzyme';
+import 'whatwg-fetch'; // fetch polyfill needed @grafana/runtime
```

Comment:
Fetch shouldn't need to be imported in here. Real fetch shouldn't be being hit.

If you want you can pull this out into a separate PR as it's mostly unrelated from this one :)